root/static/scripts/guess-case/MB/GuessCase/Handler/Base.js

```diff
@@ -34,7 +34,13 @@ class GuessCaseHandler {
 
   // Member functions
 
-  checkSpecialCase(): number {
+  checkSpecialCase(
+    /*
+     * The `inputString` argument is specified to satisfy Flow.
+     */
+    // eslint-disable-next-line no-unused-vars
+    inputString?: string,
```

Comment:
Is this a Flow bug of some sort or? Would it be better to flowissue it?

Reply:
Not a bug. The error it gives is
```js
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ root/static/scripts/guess-case/MB/GuessCase/Main.js:151:44

Cannot call handler.checkSpecialCase because no arguments are expected by function type [1]. [extra-arg]

     root/static/scripts/guess-case/MB/GuessCase/Main.js
     148│        * a special case, fetch the correct format, if the
     149│        * returned case is indeed a special case.
     150│        */
     151│       const num = handler.checkSpecialCase(inputString);
     152│       const output = handler.isSpecialCase(num)
     153│         ? handler.getSpecialCaseFormatted(inputString, num)
     154│         // if it was not a special case, start guessing

     root/static/scripts/guess-case/MB/GuessCase/Handler/Base.js
 [1]  37│   checkSpecialCase(): number {
      38│     return this.specialCaseValues.NOT_A_SPECIALCASE;
      39│   }



Found 1 error
```

Reply:
I changed this so that the argument is specified within a Flow comment instead (which eslint doesn't see).